source/includes/substitution_definitions.inc.rst

```diff
@@ -12,12 +12,23 @@
 .. |FmtAdvancedName| replace:: ``advanced``
 .. |FmtObsoleteName| replace:: ``obsolete legacy``
 
-.. |DockerHubURL| replace:: https://hub.docker.com/u/rsyslog/
+.. |FmtObsoleteDescription| replace:: ``Obsolete Format Equivalents``
+
+.. |RsyslogDockerHub| replace:: official rsyslog Docker Hub
+.. _RsyslogDockerHub: https://hub.docker.com/u/rsyslog/
+
+.. |GitHubDockerProject| replace:: rsyslog-docker project
+.. _GitHubDockerProject: https://github.com/rsyslog/rsyslog-docker
+
+.. |DockerApplianceAlpineDockerHubRepo| replace:: Alpine rsyslog appliance Docker Hub repo
+.. _DockerApplianceAlpineDockerHubRepo: https://hub.docker.com/r/rsyslog/syslog_appliance_alpine/
```

Comment:
the names are getting long ... but probably that's what simply is needed...

Reply:
Yeah. Longer names are a trade-off. Sure, they are longer, but they also make more visual sense when you see them while editing a document. With a longer reference, it almost reads like a natural part of the document and less like a programming/markup language construct. Pros and cons.

Another advantage of longer names is the ability to help differentiate between similar substitution references.

Reply:
ack - but typing too long names is a pain ;-)